### PR TITLE
ISSUE-933 WelcomeListener

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by chibenwa** [comment]
https://github.com/apache/james-project/pull/2078 was needed...

**Comment by chibenwa** [comment]
TODO document this

**Review by Arsnael — COMMENTED** [review]
Read it

**Review by quantranhong1999 — COMMENTED** [review]
nothing to add

**Review by Arsnael — APPROVED** [review]

File: docs/modules/ROOT/pages/tmail-backend/extra-components/index.adoc [hunk 1]
```diff
@@ -99,4 +99,23 @@ In order to activate it, in `rabbitmq.properties`:
...
 scheduled.consumer.reconnection.enabled=true
+....
+
+== Welcome Listener
+
+This listener will add an email in a user newly created INBOX from a predefined EML.
+
+This can be used to welcome users and avoid the "Empty INBOX newcomer syndrome".
+
+Sample comfiguration: `listeners.xml`
```
Comment (quantranhong1999) [inline docs/modules/ROOT/pages/tmail-backend/extra-components/index.adoc:110]:
```suggestion
Sample configuration: `listeners.xml`
```

**Review by quantranhong1999 — APPROVED** [review]

**Comment by quantranhong1999** [comment]
```
[INFO]  T E S T S

[INFO] -------------------------------------------------------

[INFO] Running com.linagora.tmail.mailbox.WelcomeListenerTest
```

The test is hanging on CI. Likely need to update James SHA1 with the fix.
(again the git commit id is blocking us on that. Should resolve that plugin issue soon)

**Comment by chibenwa** [comment]
> Likely need to update James SHA1 with the fix.

Exactly ;-)

**Comment by Arsnael** [comment]
The fix was included in latest sha-1 upgrade I believe, so I rebased it.

**Pull request merged**